AppCenter/AppCenter/Internals/Util/MSUtility.h

```diff
@@ -13,6 +13,11 @@
 #define MS_CLASS_NAME_WITHOUT_PREFIX [NSStringFromClass([self class]) substringFromIndex:2]
 #define MS_IS_APP_EXTENSION ([[[NSBundle mainBundle] executablePath] rangeOfString:@".appex/"].length > 0)
 #define MS_APP_MAIN_BUNDLE [NSBundle mainBundle]
+#define MS_DISPATCH_SELECTOR(result, class, selectorName, ...) ({ \
```

Comment:
Are we using this somewhere or should we just delete it for now?

Reply:
It's unused now, but I would like to keep it for future possible unknown API usage

Reply:
It's gonna be in the alert code from the other PR if we need to find it so I think we can remove it from here for now.